### PR TITLE
Merging template updates

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

This PR should take care of #54 and #51.

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -16,5 +16,9 @@ params {
   max_time = 48.h
 
   // Input data
+  // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
```
Comment (Zethson) [inline conf/test.config:19]: Can be removed.

File: conf/test_full.config [hunk 2]
```diff
@@ -11,6 +11,10 @@ params {
   config_profile_name = 'Full test profile'
   config_profile_description = 'Full test dataset to check pipeline function'
 
-  // Input data
+  // Input data for full size test
+  // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
```
Comment (Zethson) [inline conf/test_full.config:15]: Can be removed. Guess that for the full test you should also use a bigger dataset and not the same as for the small test.

But I assume that you will do this later anyways - just sayin' now.

**Comment by subwaystation** [comment]
The errors from `nf-core linting / nf-core (pull-request)` are expected, because #49 is not resolved, yet.

File: docs/output.md [hunk 3]
```diff
@@ -1,9 +1,5 @@
 # nf-core/pangenome: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/pangenome/output](https://nf-co.re/pangenome/output)
```
Comment (Zethson) [inline docs/output.md:3]: Why is this removed? No longer part of the template?

File: nextflow.config [hunk 4]
```diff
@@ -8,8 +8,17 @@
...
   // Input options
-  input = "${baseDir}/**.fa.gz"
+  input = null
+
+  // Output options
+  outdir = "./results"
+  publish_dir_mode = "copy"
+  file_name_prefix = false
+
+  // nf-core: Specify your pipeline's command line flags
```
Comment (Zethson) [inline nextflow.config:21]: Yeah, you're doing this already

**Review by Zethson — APPROVED** [review]
Super quick review. LGTM on short glance

File: docs/output.md (hunk 3 above)
Comment (subwaystation) [reply docs/output.md:3]: Yes, this came in as a part of the Template update.
